crates/biome_html_formatter/tests/quick_test.rs

```diff
@@ -0,0 +1,44 @@
+use biome_formatter::{AttributePosition, IndentStyle, LineWidth};
+use biome_formatter_test::check_reformat::CheckReformat;
+use biome_html_formatter::context::HtmlFormatOptions;
+use biome_html_formatter::{format_node, HtmlFormatLanguage};
+use biome_html_parser::parse_html;
+use biome_html_syntax::HtmlFileSource;
+
+mod language {
+    include!("language.rs");
+}
+
+#[ignore]
+#[test]
+// use this test check if your snippet prints as you wish, without using a snapshot
+fn quick_test() {
+    let src = r#"
```

Comment:
It's scope is short but you may can update the variable name `src` as below?:

- html_input
- sample_html

Reply:
Every other `quick_test` uses the name `src`, so leaving it as is would be more consistent with the rest of the codebase.

Reply:
@dyc3 i uderstood, thanks